airflow/providers/amazon/aws/links/emr.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.providers.amazon.aws.links.base_aws import BASE_AWS_CONSOLE_LINK, BaseAwsLink
+
+EMR_CLUSTER_LINK = (
+    BASE_AWS_CONSOLE_LINK + "/elasticmapreduce/home?region={region_name}#cluster-details:{job_flow_id}"
+)
+
+
+class EmrClusterLink(BaseAwsLink):
```

Comment:
If you end up following the above, this module will just contain a single 5-6 line class, I wonder if it would be better suited to live in the Operator module instead of its own module. That way we don't have the extra layer of indirection and we also save from creating new modules, directory, init files etc.

What do you think?

Reply:
It would work for Links to specific services such as EMR, Batch, ECS, EC2 however in case if we would like provide link to CloudWatch, S3 (a lot of transfers operators) it would be better store in separate place. WDYT?

Reply:
For sure, generic classes should be defined in one or more modules in `../aws/links/`, if we come to a point where we find that we need such classes. I'm just not sure an extra module for each individual (non-shared) Operator Link class is required, I think the per Operator Link classes could just live in the Operator module itself. But maybe it's clearer to have them all in once place.

I'm happy to approve either way :)

Reply:
Also I've initially think about add it for sensors however for sensors it would some side effect - for every `poke` it would write to XCom